HOW_WE_USE_GITHUB.md

```diff
@@ -154,14 +197,14 @@ Labels are defined using a scoped syntax with an optional high-level category (_
 - `[category::topic]`
 - `[category::topic-phrase]`
 
-This syntax helps with issue sorting enforcement; at minimum, both `type` and `source` labels should be specified on each issue before moving it from "`Sorting`" to "`Backlog`".
+This syntax helps with issue sorting enforcement, as it helps to ensure that sorted issues are, at minimum, categorized by type and source.
 
 There are a number of labels that have been defined for the different `conda` projects. In order to create a streamlined sorting process, label terminologies are standardized using similar (if not the same) labels.
 
 
 #### How are new labels added?
 
-New **global** labels (_i.e._, generic labels that apply equally to all `conda` repos) can be added to the `conda/infra`'s [`.github/global.yml` file][labels-global]; new **local** labels (_i.e._, labels specific to particular `conda` repos) can be added to each repository's [`.github/labels.yml`][labels-local] file. All new labels should follow the labeling syntax described in the "How are new labels defined?" section of this document.
+New **global** labels (_i.e._, generic labels that apply equally to all `conda` repos) can be added to the `conda/infrastructure`'s [`.github/global.yml` file][labels-global]; new **local** labels (_i.e._, labels specific to particular `conda` repos) can be added to each repository's [`.github/labels.yml`][labels-local] file. All new labels should follow the labeling syntax described in the "How are new labels defined?" section of this document. Global labels are synced out to all repositories as part of the [workflow sync][workflow-sync] process.
```

Comment:
The global labels config file is not synced out, they're just aggregated with any local labels when we run the sync labels workflow.

Reply:
```suggestion
New **global** labels (_i.e._, generic labels that apply equally to all `conda` repos) can be added to the `conda/infrastructure`'s [`.github/global.yml` file][labels-global]; new **local** labels (_i.e._, labels specific to particular `conda` repos) can be added to each repository's [`.github/labels.yml`][labels-local] file. All new labels should follow the labeling syntax described in the "How are new labels defined?" section of this document. Global labels are combined with any local labels and these aggregated labels are used by the `.github/workflows/labels.yml` workflow to synchronize the labels available for the repository.
```